examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -374,12 +376,30 @@
     "            delta_days = (col - item_first_timestamp) / (60*60*24)\n",
     "            gdf[column + \"_age_days\"] = delta_days * (delta_days >=0)\n",
     "        return gdf\n",
-    "           \n",
-    "    def output_column_names(self, columns):\n",
-    "        return nvt.ColumnSelector([column + \"_age_days\" for column in columns.names])\n",
     "\n",
+    "    def compute_selector(\n",
+    "        self,\n",
+    "        input_schema: Schema,\n",
+    "        selector: ColumnSelector,\n",
+    "        parents_selector: ColumnSelector,\n",
+    "        dependencies_selector: ColumnSelector,\n",
+    "    ) -> ColumnSelector:\n",
+    "        self._validate_matching_cols(input_schema, parents_selector, \"computing input selector\")\n",
+    "        return parents_selector\n",
+    "\n",
+    "    def column_mapping(self, col_selector):\n",
+    "        column_mapping = {}\n",
+    "        for col_name in col_selector.names:\n",
+    "            column_mapping[col_name + \"_age_days\"] = [col_name]\n",
```

Comment:
We might want to provide a hook for this (`def output_suffix`?), since it seems like a common pattern and it's awkward to have to copy/paste this everywhere to modify the column names.